README.md

```diff
@@ -91,6 +91,67 @@ The required parameters for this agent are "ven_name" and "vtn_url". Below is an
 Save this configuration in a JSON file in your preferred location. An example of such a configuration is saved in the
 root of this repository; the file is named `config_example1.json`
 
+# Testing
+
+If you don't have a dedicated VTN to test the VolttronOpenADR against, you can setup a local VTN instead. After setting up a local VTN, configure an VolttronOpenADRVen Agent against that local VTN and then install the agent on your VOLTTRON instance.
```

Comment:
You might describe what the steps will be doing.  I.e. the VTN will be setup on 127.0.0.1 and it will allow registration of events. etc.

Reply:
Will do.